libnl-route-3.sym

```diff
@@ -1335,6 +1335,64 @@ global:
 
 libnl_3_11 {
 global:
+	rtnl_br_vlan_gopts_alloc;
```

Comment:
btw, `3.11` is already released. Hence, the new API must be added to the `libnl_3_XX` section (where the version number is the upcoming version). See also the comment at the top of the file.